openstack/cbr/v3/policies/requests.go

```diff
@@ -0,0 +1,58 @@
+package policies
+
+import golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+
+type CreateOptsBuilder interface {
+	ToPolicyCreateMap() (map[string]interface{}, error)
+}
+
+type PolicyODCreate struct {
+	DailyBackups          int    `json:"day_backups"`
```

Comment:
`omitempty` isn't required here?

Reply:
Based on the example in the doc, it shouldn't be used